src/identify/index.js

```diff
@@ -45,22 +45,20 @@ class IdentifyService {
   /**
    * @constructor
    * @param {object} options
-   * @param {PeerStore} options.peerStore
-   * @param {ConnectionManager} options.connectionManager
+   * @param {Libp2p} options.libp2p
    * @param {Map<string, handler>} options.protocols A reference to the protocols we support
-   * @param {PeerId} options.peerId The peer running the identify service
-   * @param {{ listen: Array<Multiaddr>}} options.addresses The peer addresses
    */
   constructor (options) {
```

Comment:
Not that there are fewer params this could help simplify the `options` accessor calls in the constructor
```suggestion
  constructor ({ libp2p, protocols }) {
```